scripts/world_bank/wdi/schema_csvs/WorldBankIndicators_prod.csv

```diff
@@ -14,22 +14,22 @@ SE.TER.CUAT.ST.ZS,,,"Educational attainment, at least completed short-cycle tert
 SH.STA.OWGH.FE.ZS,,,"Prevalence of overweight, weight for height, female (% of children under 5)","Prevalence of overweight, female, is the percentage of girls under age 5 whose weight for height is more than two standard deviations above the median for the international reference population of the corresponding age as established by the WHO's new child growth standards released in 2006.","UNICEF, WHO, World Bank: Joint child malnutrition estimates (JME). Aggregation is based on UNICEF, WHO, and the World Bank harmonized dataset (adjusted, comparable data) and methodology.",Person,measuredValue,count,healthBehavior,Overweight,age,YearsUpto4,gender,Female,Count_Person_Upto4Years_Female,,100,,
```

Comment:
pls set unit to Percent for scalingFactor:100

Reply:
Added